client/src/css/stylesheet.scss

```diff
@@ -0,0 +1,211 @@
+@font-face {
+  font-family: "Open Sans";
+  src: url("../fonts/OpenSans-Regular.ttf") format("truetype");
```

Comment:
Can you elaborate I'm not super familiar with serving webpages. This was the only way I could get it to work. `local()` wasn't loading it.